scripts/zones/Misareaux_Coast/npcs/HomePoint#1.lua

```diff
@@ -0,0 +1,55 @@
+-----------------------------------
+-- Area: Misareaux Coast
+-- NPC:  HomePoint#1
+-- @pos -65 -17.5 563 25
+-----------------------------------
+
+package.loaded["scripts/zones/Misareaux_Coast/TextIDs"] = nil;
+
+require("scripts/globals/settings");
+require("scripts/zones/Misareaux_Coast/TextIDs");
+require("scripts/globals/homepoint");
+
+-----------------------------------
+-- onTrade Action
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end; 
+
+-----------------------------------
+-- onTrigger Action
+-----------------------------------
+
+function onTrigger(player,npc)
+
+	homepointMenu( player, 0x21fd, 25);
+end; 
+
+-----------------------------------
+-- onEventUpdate
+-----------------------------------
+
+function onEventUpdate(player,csid,option)
+	--printf("CSID: %u",csid);
+	--printf("RESULT: %u",option);
+end;
+
+-----------------------------------
+-- onEventFinish
+-----------------------------------
+
+function onEventFinish(player,csid,option)
+	--printf("CSID: %u",csid);
+	--printf("RESULT: %u",option);
+
```

Comment:
For future reference: Darkstar started using a 4 space standard instead of tabs in 1 file and spaces in another.